### PR TITLE
no property

### PR DESCRIPTION
**Pull request by jwang173**

### CODE REVIEW

**Review by linhhuehuynh — APPROVED** [review]

**Review by OseankingO — APPROVED** [review]

**Review by yuz203 — APPROVED** [review]

**Review by yuz203 — APPROVED** [review]

File: ResourceManagement/src/main/java/com/groupproject/resourcemanagement/entity/ProjectResourceEntity.java [hunk 1]
```diff
@@ -47,7 +47,7 @@ public int hashCode() {
     }
 
     @ManyToOne
-    @JoinColumn(name = "project_id", referencedColumnName = "project_id", nullable = false)
```
Comment (jwang173) [inline ResourceManagement/src/main/java/com/groupproject/resourcemanagement/entity/ProjectResourceEntity.java:50]: no insertable false

File: ResourceManagement/src/main/java/com/groupproject/resourcemanagement/entity/ProjectColumnEntity.java [hunk 2]
```diff
@@ -4,7 +4,7 @@
 import java.util.Objects;
 
 @Entity
-@Table(name = "project_column", schema = "mydb", catalog = "")
```
Comment (jwang173) [inline ResourceManagement/src/main/java/com/groupproject/resourcemanagement/entity/ProjectColumnEntity.java:7]: no database changed

File: ResourceManagement/src/main/java/com/groupproject/resourcemanagement/entity/ProjectColumnEntity.java [hunk 3]
```diff
@@ -23,7 +23,7 @@ public void setProjectColumnId(Integer projectColumnId) {
     }
 
     @Basic
-    @Column(name = "project_id")
```
Comment (jwang173) [inline ResourceManagement/src/main/java/com/groupproject/resourcemanagement/entity/ProjectColumnEntity.java:26]: no insertable changed

File: ResourceManagement/src/main/java/com/groupproject/resourcemanagement/entity/ProjectEntity.java [hunk 4]
```diff
@@ -6,7 +6,7 @@
 import java.util.Objects;
 
 @Entity
-@Table(name = "project", schema = "mydb", catalog = "")
```
Comment (jwang173) [inline ResourceManagement/src/main/java/com/groupproject/resourcemanagement/entity/ProjectEntity.java:9]: no db changed

File: ResourceManagement/src/main/java/com/groupproject/resourcemanagement/entity/ProjectEntity.java [hunk 5]
```diff
@@ -85,7 +85,7 @@ public int hashCode() {
     }
 
     @ManyToOne
-    @JoinColumn(name = "user_id", referencedColumnName = "user_id", nullable = false)
```
Comment (jwang173) [inline ResourceManagement/src/main/java/com/groupproject/resourcemanagement/entity/ProjectEntity.java:88]: no insertable changed

File: ResourceManagement/src/main/java/com/groupproject/resourcemanagement/entity/ProjectResourceEntity.java [hunk 6]
```diff
@@ -57,7 +57,7 @@ public void setProjectByProjectId(ProjectEntity projectByProjectId) {
     }
 
     @ManyToOne
-    @JoinColumn(name = "resource_id", referencedColumnName = "resource_id", nullable = false)
```
Comment (jwang173) [inline ResourceManagement/src/main/java/com/groupproject/resourcemanagement/entity/ProjectResourceEntity.java:60]: no insertable

File: ResourceManagement/src/main/java/com/groupproject/resourcemanagement/entity/ResourceEntity.java [hunk 7]
```diff
@@ -106,7 +106,7 @@ public void setProjectResourcesByResourceId(Collection<ProjectResourceEntity> pr
     }
 
     @ManyToOne
-    @JoinColumn(name = "parent_resource_id", referencedColumnName = "resource_id")
```
Comment (jwang173) [inline ResourceManagement/src/main/java/com/groupproject/resourcemanagement/entity/ResourceEntity.java:109]: no insertable changed

**Review by yuz203 — APPROVED** [review]

**Review by linhhuehuynh — APPROVED** [review]

**Review by linhhuehuynh — APPROVED** [review]

**Comment by jwang173** [comment]
3 deleted

**Comment by jwang173** [comment]
reopen

**Pull request closed without merging**